rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt

```diff
@@ -262,6 +262,21 @@ data class ZSubStmt(
         return emptyList()
     }
 }
+data class SubStmt(
+    val factor1: Expression?,
```

Comment:
I would use more expressive names then factor1 and factor2, like subtrahend and minuend

Reply:
Also "left" would be a better name, in my opinion

Reply:
Done